### DISCUSSION TITLE
DRC error in detailed routing - Short

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#3217] [General]
What is detailed routing trying to do here? The grey area is a macro.

Is the detailed route simply spilling onto the macro because it can't find a valid place to put the wire?

![image](https://user-images.githubusercontent.com/2798822/233313803-be73a31c-255b-4857-acb3-d304cb37db11.png)

![image](https://user-images.githubusercontent.com/2798822/233314427-9f2f4378-db70-4f2f-bc54-c23118c5e2b9.png)

**Comment by oharboe** [comment]
Since no-one answers, I'm thinking I guessed right :-)

**Comment by maliberty** [comment]
Most likely (assuming the layers are blocked and this isn't just available resource).
